src/main/interface/settings.c

```diff
@@ -684,6 +684,9 @@ const clivalue_t valueTable[] = {
     { "dshot_idle_value",           VAR_UINT16  | MASTER_VALUE, .config.minmax = { 0, 2000 }, PG_MOTOR_CONFIG, offsetof(motorConfig_t, digitalIdleOffsetValue) },
 #ifdef USE_DSHOT_DMAR
     { "dshot_burst",                VAR_UINT8  | MASTER_VALUE | MODE_LOOKUP, .config.lookup = { TABLE_OFF_ON }, PG_MOTOR_CONFIG, offsetof(motorConfig_t, dev.useBurstDshot) },
+#ifdef USE_DSHOT_TELEMETRY
```

Comment:
We should define this in `src/main/target/common_fc_pre.h`, so it actually gets built on CI. Does this fit onto all F3s? If not, we can just enable it for > 256 kB flash, and enable on F3 on a target by target basis.

Reply:
Agreed.